src/vs/workbench/services/themes/common/colorThemeData.ts

```diff
@@ -426,10 +425,53 @@ export class ColorThemeData implements IWorkbenchColorTheme {
 		this.textMateThemingRules = undefined;
 	}
 
+	public isThemeScope(key: string): boolean {
```

Comment:
This ideally should be a type assert using a template literal type from `workbenchThemeService` module